foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -94,6 +94,15 @@
             "includes": ["src/**/*.java"]
           }
         ]
+      },
+      {
+        "name": "extensions-security-projects-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_8_0.ExtensionsSecurityProjectsMigration",
+        "fileSets": [
+          {
+            "includes": ["src/**/*.java"]
+          }
+        ]
```

Comment:
S: This and some of the migrations above should be in the 1.8.0 group.

Reply:
Moved the migrations to the 1.8.0 group.